src/yb/rocksdb/util/thread_posix.cc

```diff
@@ -176,7 +176,11 @@ void ThreadPool::StartBGThreads() {
     CHECK_OK(yb::Thread::Create(
         category_name, std::move(thread_name),
         [this, tid]() { this->BGThread(tid); }, &thread));
-    bg_wait_states_.push_back(std::make_shared<yb::util::WaitStateInfo>());
+    bg_wait_states_.push_back(std::make_shared<yb::util::WaitStateInfo>(
+                                yb::util::AUHMetadata{
+                                    .top_level_node_id = "localhost",
+                                    .top_level_request_id = {0, 2},
+                                    .query_id = -2}));
```

Comment:
Re: discussion from previous PR

My initial thought was that these metadata fields are going to be the same for each thread, so instead of copying it for each thread, we can just populate it when we are sending this info to PG. But this will only make sense for std::string because we are not allocating memory. So now I think it's fine to update the metadata here.